proto/storage/cve.proto

```diff
@@ -161,6 +164,37 @@ message ImageCVE {
   CvssScoreVersion nvd_score_version = 12;
 }
 
+message ImageCVEV2 {
+  // keeping id similar construction for now during investigation.  It will include component and index
+  // within the component
+  string id = 1; // @gotags: search:"CVE ID,hidden" sql:"pk,id"
```

Comment:
One note on IDs for the reviewers.  I reserve the right to switch these to hashes as we get further along.  Building them similar to how they are built down decreases the chance of unexpected ripple effects.  Once implementation is more complete I can re-evaluate using a string vs hash.  I think for the most part we can all probably agree a hash OR building a UUID would be preferable to a string concatenation.